### DISCUSSION TITLE
Does a site license have downgrade rights?

### DISCUSSION THREAD

**Discussion by aguywhoworksinit** [usnistgov/REFPROP-issues#605] [Q&A]
The title pretty much says it all, specifically though. If I have a site license for REFPROP 10, can my organization intall REFPROP 9 or REFPROP 8? Are there downgrade rights, and if so, how many versions?

**Comment by ianhbell** [comment]
No, there are no downgrade rights. Besides, REFPROP 10 is a big improvement over REFPROP 8
